### PR TITLE
Crf

### PR DESCRIPTION
**Pull request by andrewfraser73**
basic change request viewer for just a few fields

### CODE REVIEW

File: src/Resources/ChangeRequestResource.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    using System;
+
+    public class ChangeRequestResource
+    {
+        public string DocumentType { get; set; }
+
+        public int DocumentNumber { get; set; }
+
+        public DateTime DateEntered { get; set; }
```
Comment (richardiphillips) [inline src/Resources/ChangeRequestResource.cs:11]: should be iso string in a resource

**Review by richardiphillips — APPROVED** [review]
could start the integration tests before merging

**Comment by andrewfraser73** [comment]
> could start the integration tests before merging

actually had already written them but for some reason hadn't pushed them up

File: tests/Integration/Integration.Tests/ChangeRequestModuleTests/WhenGettingChangeRequest.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        [Test]
+        public void ShouldCallRepository()
+        {
+            this.Repository.Received().FindById(1);
+        }
+
+        [Test]
+        public void ShouldReturnJsonBody()
+        {
+            var result = this.Response.DeserializeBody<ChangeRequestResource>;
```
Comment (richardiphillips) [inline tests/Integration/Integration.Tests/ChangeRequestModuleTests/WhenGettingChangeRequest.cs:60]: there's no test in here

**Pull request merged**